workflows/get_data_irmng.py

```diff
@@ -0,0 +1,168 @@
+import pathlib
```

Comment:
Is this file still needed?

Reply:
No, get_data_irmng.py file is not required now